tests/commands/test_commands.py

```diff
@@ -392,24 +391,67 @@ def mock_write(tx, mode):
     args = Namespace(action='new',
                      public_key='CJxdItf4lz2PwEf4SmYNAu/c/VpmX39JEgC5YpH7fxg=',
                      power=1,
-                     node_id='12345',
+                     node_id='fb7140f03a4ffad899fabbbf655b97e0321add66',
                      sk=priv_validator_path,
                      config={})
     resp = run_upsert_validator_new(args, b)
 
     assert b.get_transaction(resp)
 
 
+@pytest.mark.tendermint
+@pytest.mark.bdb
+def test_upsert_validator_new_invalid_election(b, priv_validator_path, user_sk):
+    from bigchaindb.commands.bigchaindb import run_upsert_validator_new
+
+    b.get_validators = mock_get
+    args = Namespace(action='new',
+                     public_key='CJxdItf4lz2PwEf4SmYNAu/c/VpmX39JEgC5YpH7fxg=',
+                     power=10,
+                     node_id='fb7140f03a4ffad899fabbbf655b97e0321add66',
+                     sk=priv_validator_path,
+                     config={})
+    assert not run_upsert_validator_new(args, b)
```

Comment:
If we return a particular response or raise a particular exception we can then test for particular outcomes. Currently, the tests do not guarantee the correct outcome (now expressed as logs) for these cases - they only guarantee a failure occurs.